src/config/ConfigOptions.ts

```diff
@@ -0,0 +1,14 @@
+export interface IConfigOptions {
+  posthog?: {
+    api_key: string;
+  };
+  sentry?: {
+    dns: string;
+    environment: string;
+  };
+  rageshake?: {
+    submit_url: string;
+  };
+}
+
+export const DEFAULT: IConfigOptions = {};
```

Comment:
```suggestion
export const DEFAULT_CONFIG: IConfigOptions = {};
```

To be more descriptive